tests/LineBot.Tests/RichMenuRequestTests/TheConvertMethodTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
```

Comment:
This file should be renamed to `tests/LineBot.Tests/RichMenu/RichMenuTests/TheConvertMethod.cs`.